docs/concepts-broker-load-balancing-quick-start.md

```diff
@@ -0,0 +1,594 @@
+:::
+
+### Step 6. Unload bundles
+
+You can verify if a bundle has been unloaded using the metric [pulsar_lb_unload_bundle_total](./reference-metrics.md#load-balancing). This method is only supported in the automatic way.
+
+:::tip
+
+- The automatic bundle unloading is **enabled by default**. To disable it, update [loadBalancerSheddingEnabled](https://github.com/apache/pulsar/blob/69d7a2bf14555f11a716a9545c5cf391d8179a27/conf/broker.conf#L1259C14-L1259C14) to false in the broker.conf file. 
+
+- The **default** bundle unloading strategy is TransferShedder (for the extensible broker load balancer) or ThresholdShedder (for the modular broker load balancer). You can change it to other [bundle unloading strategies](./concepts-broker-load-balancing-concepts.md#bundle-unloading-strategies) by updating [loadBalancerLoadSheddingStrategy](https://github.com/apache/pulsar/blob/69d7a2bf14555f11a716a9545c5cf391d8179a27/conf/broker.conf#L1324C52-L1324C52) in the broker.conf file. 
+
+- For bundle unloading conditions, you can set more configurations in the broker.conf file. 
+:::
+
+## Configure broker load balancer to run manually
+
+If you want to use broker load balancing manually, follow the steps below.
+
+### Step 1. Start ZooKeeper, bookie, and broker
+
+This is the same as [Step 1. Start ZooKeeper, bookie, and broker](#step-1-start-zookeeper-bookie-and-broker) in the automatic method.
+
+### Step 2. Start a Pulsar cluster
+
+This is the same as [Step 2. Start a Pulsar cluster](#step-2-start-a-pulsar-cluster) in the automatic method.
+
+### Step 3. Create a topic
+
+3.1 Enter the container of broker 1.
+
+**Input**
+
+```bash
+docker exec -it broker-1 bash
+```
+
+**Output**
+
+If there is no output, the operation is successful.
+
+3.2 Create a partitioned topic named topic-manual with 20 partitions under public/default.
+
+**Input**
+
+```bash
+bin/pulsar-admin topics create-partitioned-topic persistent://public/default/topic-manual -p 20
+```
+
+**Output**
+
+If there is no output, the operation is successful.
+
+### Step 4. Split bundles
+
+4.1 List all bundles.
+
+**Input**
+
+```bash
+bin/pulsar-admin namespaces bundles public/default
+```
+
+**Output**
+
+The output shows that the number of bundles is 17.
+
+```bash
+{
+  "boundaries" : [ "0x00000000", "0x08000000", "0x10000000", "0x20000000", "0x30000000", "0x40000000", "0x50000000", "0x60000000", "0x70000000", "0x80000000", "0x90000000", "0xa0000000", "0xb0000000", "0xc0000000", "0xd0000000", "0xe0000000", "0xf0000000", "0xffffffff" ],
+  "numBundles" : 17
+}
+```
+
+4.2 Specify the range `public/default/0x00000000_0x8000000` to be split.
+
+**Input**
+
+```bash
+bin/pulsar-admin namespaces split-bundle --bundle 0x00000000_0x10000000 public/default
```

Comment:
We recently added an option, `-d` to specify the destination owner broker. It could be useful to explain this too.

Reply:
OK, added